### ISSUE TITLE
PMID:25356590

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3708]
Multifaceted genome control by Set1 Dependent and Independent of H3K4 methylation and the Set1C/COMPASS complex.

Deletion of specific components of COMPASS affects the integrity of the complex and sta- bility of Set1 [31,48,102–105]. The deletion of Spp1, Swd1, Swd2, and Swd3 significantly lowers the protein levels of Set1 [48].

**Comment by ValWood** [comment]
Set1 proteins were barely detectable in swd1Δ and swd3Δ and were noticeably reduced in swd2Δ and spp1Δ mutants, but were minimally affected by the loss of Ash2, Sdc1, and Shg1 [48].

**Comment by ValWood** [comment]
This is the one I sent a session for. No hurry for this, I'm just trying to  assign some tickets

**Comment by PCarme** [comment]
I have curated this one, it is now awaiting approval. 
I would appreciate if you could double check this one, I am not entirely sure of all the FYPO terms I have used/suggested to annotate the phenotypes, and I feel like my GO annotations might be missing some elements...

**Comment by ValWood** [comment]
I think this looks fine.
Could also add
retrotransposon silencing by heterochromatin formation
(or a descendant?)

but this term is a bit strange, I think there are still issues in the heterochromatin formation processes, but at least if we capture it  the annotation will "move around" with any corrections.

**Comment by PCarme** [comment]
I think I have already annotated to this term, or to a similar one, no ?

**Comment by ValWood** [comment]
Yes sorry, I noted it because I wanted to report a GO missing parent, but the annotation was present, and correct!
